### DISCUSSION TITLE
Following install process video - install fails

### DISCUSSION THREAD

**Discussion by orbitcoms** [Mair/esp32-course#83] [Course Qeustions] [answered]
I have removed VSCODE and esp-idf from my drive and proceeded to reinstall following your "Installing the esp-idf" video.

When I navigate to the Windows installer from espressif , the options are slightly different so I select the online version at top (called universal online installer). However, the install went for a while then crash with attached error.

![install error](https://user-images.githubusercontent.com/41096656/164141628-a77e7d0c-456f-4425-b357-8d4ce4283f81.JPG)

**Comment by Mair** [comment]
@orbitcoms 
Not a lot to go on from the error message. Have you tried reviewing the logs?

**Comment by orbitcoms** [comment, marked as answer]
I unstalled and reinstalled esp-idf using the extension and "Advanced" mode and now it works again.
Been having these "strange" issues periodcally, both on my old PC and this new one.

So, I loaded the projects up in Visual Studio 2022 with VisualGDB installed and they all run fine. I might stick with this toll for a while as it seems consistent.
